src/OrchardCore.Modules/OrchardCore.Queries/Razor/ContentQueryOrchardRazorHelperExtensions.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using OrchardCore.DisplayManagement.Razor;
+using OrchardCore.Queries;
+
+namespace OrchardCore.ContentManagement
+{
+    public static class ContentQueryOrchardRazorHelperExtensions
```

Comment:
@sebastienros I know you said that this extension should go in ContentManagement.Abstractions, but that would require CM.Abs to have a reference to OC.Queries